### PR TITLE
Add how-to to connect a custom Grafana to user worload monitoring

### PR DESCRIPTION
**Pull request by glrf**
## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+Adding a single data source for multiple project isn't supported.
+
+. Give the service account permission to access the metrics of the project.
+For that you need to grant the service account permission to view pods in the target namespace.
++
+[source,shell]
+----
+APP_NAMESPACE="my-app"<1>
+
+kubectl -n "${APP_NAMESPACE}" create rolebinding grafana-viewer --clusterrole=appuio:metrics-reader --serviceaccount="${GRAFANA_NAMESPACE}:grafana-viewer"
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc:65]: Where's the clusterrole `appuio:metrics-reader` managed?

File: docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+----
+TOKEN=$(kubectl -n "${GRAFANA_NAMESPACE}" get secret grafana-viewer -ojsonpath={.data.token} | base64 -d)
+CA=$(kubectl -n "${GRAFANA_NAMESPACE}" get secret grafana-viewer -ojsonpath={.data."service-ca\.crt"} | base64 -d)
+----
+
+
+== Add Data Source for Project
+
+NOTE: You'll need to add each project as a separate data source.
+Adding a single data source for multiple project isn't supported.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc:56]:
```suggestion
Adding a single data source for multiple projects isn't supported.
```

**Review by simu — APPROVED** [review]
LGTM overall

File: docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc (hunk 1 above)
Comment (glrf) [reply docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc:65]: Handled in: https://github.com/appuio/component-appuio-cloud/pull/135

File: docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+!===
+
+|===
++
+[TIP]
+====
+Grafana defaults to the HTTP method `POST`, however we found inconsistencies when using it to access metrics of a single project. 
+We recommend to fall back to `GET`.
+
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc:152]:
```suggestion
====

[NOTE]
====
```

File: docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+!===
+
+! Custom query parameters
+! `namespace=${APP_NAMESPACE}`
+
+!===
+
+|===
++
+[TIP]
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc:148]:
```suggestion
[IMPORTANT]
```

File: docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+!===
+
+| *TLS/SSL Auth Details*
+|
+
+[cols="1,3"]
+!===
+
+! CA Cert
+! `$CA`
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc:109]: This one I do not understand. Even in combination with looking at the screenshots. Am I supposed to literally insert `$CA` into the input field?

**Review by corvus-ch — CHANGES_REQUESTED** [review]
Please clarify how the CA must be set exactly.

**Review by corvus-ch — APPROVED** [review]

File: docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc (hunk 5 above)
Comment (corvus-ch) [reply docs/modules/ROOT/pages/how-to/use-custom-grafana.adoc]: Sorry, I had a know in my brain.

**Pull request merged**